test/parallel/test-process-cpuUsage.js

```diff
@@ -33,27 +33,57 @@ for (let i = 0; i < 10; i++) {
   assert(diffUsage.system >= 0);
 }
 
+const invalidUserArgument = new RegExp('^TypeError: value of user ' +
```

Comment:
I believe the official style is what I suggested (although I can't remember if the indentation on the next line is supposed to be 2 or 4 spaces).